indico/migrations/versions/20210705_1635_1cec32e42f65_add_reviewed_dt_to_paper_revisions.py

```diff
@@ -0,0 +1,39 @@
+"""Add reviewed_dt to paper revisions
+
+Revision ID: 1cec32e42f65
+Revises: 420195768776
+Create Date: 2021-07-05 16:35:54.667174
+"""
+import sqlalchemy as sa
+from alembic import op
+
+from indico.core.db.sqlalchemy import UTCDateTime
+from indico.modules.events.editing.models.revisions import FinalRevisionState
```

Comment:
alembic revisions usually shouldn't import stuff containing data (remember, a revision may run much later when the underlying enum changed in some way or was removed altogether). just hardcode the value of the enum member you use